ci-operator/step-registry/cucushift/upgrade/eus/pause-worker-mcp/cucushift-upgrade-eus-pause-worker-mcp-ref.yaml

```
@@ -8,6 +8,10 @@ ref:
   grace_period: 10m
   commands: cucushift-upgrade-eus-pause-worker-mcp-commands.sh
   cli: latest
+  env:
+  - name: PAUSED_MCP_LABEL
```

```suggestion
  - name: MCP_NAME
```

I think users just need to input MCP name, not related to node label stuff.

---

I want keep cucushift-upgrade-eus-pause-worker-mcp and cucushift-upgrade-eus-unpause-worker-mcp have same environment variables. so I can rename it to `PAUSED_MCP_NAME` in all steps, thoughts?